src/components/RangeSlider/UniversalSlider.tsx

```diff
@@ -134,7 +131,7 @@ const UniversalSliderDumb: React.FC<UniversalSliderProps<UniversalValue>> = ({
       {...restProps}
       {...(disabled ? {} : { onStart, onMove, onEnd })}
       vkuiClass={classNames(
-        getClassName("Slider", platform),
+        "Slider",
         `Slider--sizeY-${sizeY}`,
```

Comment:
Коммент "TODO v5.0.0 Новая адаптивность"?

Reply:
Но я же не трогаю адаптивность

Reply:
В v5.0.0 нужно заменить на

```suggestion
        getSizeYClassName(`Slider--sizeY-${sizeY}`),
```

Reply:
Не вижу каких-то больших проблем в поиске такого и массовой замене
<img width="1440" alt="Снимок экрана 2022-06-14 в 14 08 17" src="https://user-images.githubusercontent.com/14944123/173563790-394f929a-4390-41f5-aab3-54a11045c96e.png">